### PR TITLE
Feature multi field

### PR DESCRIPTION
**Pull request by Akasch**
This adds the possibility to render Djangos MultValueFields.

The default rendering is as an input-group, but it is possible to render the parts separately as normal fields.

### CODE REVIEW

File: django-spectre-css/templates/spectre-css/render-multi-field-part.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{% load widget_tweaks_extras %}
+{% if bare %}
+    {% render_multi_field field index class+=additional_class class+=field_class id=field_id %}
+{% else %}
+    <div class="form-group">
+        <label class="form-label" for="{{ field_id }}">{{ label_text }}{# field.label #}</label>
```
Comment (wichmannpas) [inline django-spectre-css/templates/spectre-css/render-multi-field-part.html]: We could remove that obsolete commented-out code completely

File: django-spectre-css/templates/spectre-css/render-multi-field-part.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                {% render_multi_field field index class+=additional_class class+="is-error" class+=field_class id=field_id %}
+                {% include "spectre-css/render_errors.html" %}
+            {% else %}
+                {% render_multi_field field index class+=additional_class class+=field_class class+="is-success" id=field_id %}
+            {% endif %}
+        {% else %}
+            {% render_multi_field field index class+=additional_class class+=field_class id=field_id %}
+        {% endif %}
+        {% if field.index.help_text %}
+            <small class="form-text text-muted">{{ field.index.help_text|safe }}</small>
```
Comment (wichmannpas) [inline django-spectre-css/templates/spectre-css/render-multi-field-part.html:18]: We should make the warning in the README a bit more visible and not just place it within the example section.

File: django-spectre-css/templatetags/spectre_css.py [hunk 3]
```diff
@@ -19,14 +19,19 @@ def is_select(field):
     return isinstance(field.field.widget, Select)
 
 
+@register.filter
+def is_multi_field(field: BoundField):
+    return "fields" in field.field.__dict__
```
Comment (wichmannpas) [inline django-spectre-css/templatetags/spectre_css.py]: Is there no cleaner way to check for a MultiField? E.g., something like issubclass or `isinstance`?

File: django-spectre-css/templatetags/spectre_css.py [hunk 4]
```diff
@@ -79,3 +84,30 @@ def render_form_radio(field: BoundField):
 @register.inclusion_tag('spectre-css/form.html')
 def render_form(form: Form):
     return {'form': form}
+
+
+@register.inclusion_tag('spectre-css/form-haeder.html')
```
Comment (wichmannpas) [inline django-spectre-css/templatetags/spectre_css.py]: Typo in file name (haeder -> header)

File: django-spectre-css/templatetags/widget_tweaks_extras.py [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        self.index = index
+
+    def render(self, context):
+        index = self.index.resolve(context)
+        bounded_field = self.field.resolve(context)
+        field = bounded_field.field.fields[index]
+        widget = bounded_field.field.widget.widgets[index]
+
+        attrs = widget.attrs.copy()
+        print(attrs)
```
Comment (wichmannpas) [inline django-spectre-css/templatetags/widget_tweaks_extras.py]: This should be removed, should it not?

**Review by wichmannpas — COMMENTED** [review]
Looks good, only some very few minor remarks.

File: django-spectre-css/templates/spectre-css/render-multi-field-part.html (hunk 2 above)
Comment (Akasch) [reply django-spectre-css/templates/spectre-css/render-multi-field-part.html:18]: I have moved the notice to its own section and added that one should not put user generated input into it.

File: django-spectre-css/templatetags/spectre_css.py [hunk 6]
```diff
@@ -21,7 +21,7 @@ def is_select(field):
 
 @register.filter
 def is_multi_field(field: BoundField):
-    return "fields" in field.field.__dict__
+    return issubclass(type(field), MultiValueField)
```
Comment (wichmannpas) [inline django-spectre-css/templatetags/spectre_css.py]: Would `isinstance(field, MultiValueField)` work and be cleaner?

Comment (Akasch) [reply django-spectre-css/templatetags/spectre_css.py]: yes it was possible...

**Pull request merged**